ios/Button.swift

```diff
@@ -56,18 +57,35 @@ class Button : RCTView {
   override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
     animateTapStart(
       duration: duration,
-      scale: scaleTo,
-      useHaptic: enableHapticFeedback ? hapticType: nil
+      scale: scaleTo
     )
+    
+    if self.enableHapticFeedback {
+      DispatchQueue.main.asyncAfter(deadline: .now() + 0.2) {
```

Comment:
Can we move the 0.2 to a var at the top?
Maybe something like:
```
@objc var hapticFeedbackDelay: TimeInterval = 0.2
```

Reply:
I discussed it with @christianbaroni and we're about to remove this delay altogether